aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/plugins/question_list.html

```diff
@@ -1,3 +1,5 @@
+{% load i18n %}
```

Comment:
we had once the discussion that performance is not affected, we use it as good practice to add at least i18n which it is also the case in aldryn-newsblog